barretenberg/cpp/pil/vm2/calldata.pil

```diff
@@ -38,7 +38,13 @@ include "calldata_hashing.pil";
 // -----------+-----------+----------+------------+----------+----------+----------+-------------+-------+-------+------------+---------+---------------+
 //     0      |     1     |     2    |      1     |    sep   |   0x111  |   0x222  |  cd_hash_0  |   0   |   1   |     2      |    0    |       4       |
 //     3      |     4     |     5    |      1     |   0x333  |   0x444  |     0    |  cd_hash_0  |   1   |   0   |     1      |    1    |       4       |
-
+//
+// Usage:
+// sel {
+//     index /*offset plus one*/, context_id, value
```

Comment:
offset is only mentioned here in this file so it's not so clear what that means, can we replace this comment by just saying `index starts at 1`?

I think this should just reference the helpful note on line 22 or on line 56 on why we start at 1-indexed e.g.
`This is the only case in which index = 0 and sel = 1. Note that no lookups should access this trace`